giakhang94/maylanh
# [REVIEW] refactor: [FE] #6 -use vite

#6 

## Purpose of this branch 
- Migrate from CRA to Vite ([link](https://www.freecodecamp.org/news/how-to-migrate-from-create-react-app-to-vite/))
- Add path alias @/* to lite up long imports and use it in App.tsx
- Do minor fix for axios interceptor: fetch to BE now use exactly BE URL (`localhost::5000` instead of `localhost::3000`)
```tsx
import {
  AddService,
  AllService,
  Dashboard,
  Home,
  Login,
  Order,
  Register,
  ShareLayout,
} from "@/pages";
```

## How to test
1. checkout the branch
2. run `npm install` for both BE and FE
3. run `npm start` from root of BE

Please notice that there is a change in axios URL, please help me check if the app is working properly. Thanks
